packages/generate/src/edgeql-js/generateIndex.ts

```diff
@@ -122,7 +127,10 @@ export function generateIndex(params: GeneratorParams) {
         allowFileExt: true,
       });
 
-      index.writeln([r`${genutil.quote(moduleName)}: _${internalName},`]);
+      const valueStr = defaultSpreadTypes.has(moduleName)
+        ? `{ ..._${internalName}, ..._default.${moduleName} }`
```

Comment:
Should this be frozen? Following suit of https://github.com/edgedb/edgedb-js/blob/d99c022a02b746585af7b706d3f7ba9fadc81d22/packages/generate/src/syntax/path.ts#L433-L433